src/main/java/org/jabref/gui/util/DefaultTaskExecutor.java

```diff
@@ -122,6 +122,9 @@ public static void runInJavaFXThread(Runnable runnable) {
         return scheduledExecutor.schedule(getJavaFXTask(task), delay, unit);
     }
 
+    /**
+     * Shuts everything down. Upon termination, this method returns.
```

Comment:
add to interface

Reply:
Can't do - Added a comment to TaskExecutor